blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -1,123 +1,245 @@
+	const shouldLazyLoad = lazyLoad && !isAdmin;
 	if (shouldLazyLoad && isServerSide()) {
 		return null;
 	}
-	return (
+
+	const displayDate = content?.display_date;
+	const formattedDate = Date.parse(displayDate)
+		? localizeDateTime(new Date(displayDate), dateTimeFormat, language, timeZone)
+		: "";
+
+	const { display: labelDisplay, url: labelUrl, text: labelText } = content?.label?.basic || {};
+
+	const shouldUseLabel = !!labelDisplay;
+
+	const { _id: sectionUrl, name: sectionText } =
+		content?.websites?.[arcSite]?.website_section || {};
+
+	// Default to websites object data
+	let [overlineText, overlineUrl] = [sectionText, sectionUrl];
+
+	if (content?.owner?.sponsored) {
+		const phrases = getTranslatedPhrases(locale || "en");
+		overlineText = content?.label?.basic?.text || phrases.t("overline.sponsored-content");
+		overlineUrl = null;
+	} else if (shouldUseLabel) {
+		[overlineText, overlineUrl] = [labelText, labelUrl];
+	}
+
+	const contentUrl = content?.websites?.[arcSite]?.website_url;
+	const contentHeading = showHeadline ? content?.headlines?.basic : null;
+	const contentDescription = showDescription ? content?.description?.basic : null;
+	const promoImageURL = imageOverrideURL || extractImageFromStory(content);
+	const imageSearchField = imageOverrideURL ? "imageOverrideURL" : "imageURL";
+	const embedMarkup = playVideoInPlace && extractVideoEmbedFromStory(content);
+
+	const MediaImage = () =>
+		showImage ? (
+			<Conditional component={Link} condition={contentUrl} href={contentUrl}>
+				<Image
+					alt={content?.headlines?.basic}
+					src={promoImageURL || fallbackImage}
+					searchableField
+					data-aspect-ratio={imageRatio?.replace(":", "/")}
+				/>
+			</Conditional>
+		) : null;
+
+	return showOverline ||
+		contentHeading ||
+		showImage ||
+		contentDescription ||
+		showByline ||
+		showDate ? (
 		<LazyLoad enabled={shouldLazyLoad}>
-			<ExtraLargePromoItem customFields={customFields} arcSite={arcSite} />
+			<article className={BLOCK_CLASS_NAME}>
+				{showOverline ? <Overline href={overlineUrl}>{overlineText}</Overline> : null}
+				{contentHeading || showImage || contentDescription || showByline || showDate ? (
+					<Stack>
+						{contentHeading ? (
+							<HeadingSection>
+								<Heading>
+									<Conditional component={Link} condition={contentUrl} href={contentUrl}>
+										{contentHeading}
+									</Conditional>
+								</Heading>
+							</HeadingSection>
+						) : null}
+						{embedMarkup || showImage ? (
+							<MediaItem {...searchableField(imageSearchField)} suppressContentEditableWarning>
+								{embedMarkup ? <Video embedMarkup={embedMarkup} /> : <MediaImage />}
+							</MediaItem>
+						) : null}
+						{contentDescription ? <Paragraph>{contentDescription}</Paragraph> : null}
+						{showByline || showDate ? (
+							<Attribution>
+								<Join separator={Separator}>
+									{showByline ? formatAuthors(content?.credits?.by) : null}
```

Comment:
You are missing the translations for the `By` and the `and` part of the attribution

Reply:
Added this miss back in and globalized the translations